### DISCUSSION TITLE
Translation in configuration of orders statuses labels

### DISCUSSION THREAD

**Discussion by gausoft** [lunarphp/lunar#1281] [Q&A]
Hi, is there a way to translate order labels?

![Screen Shot 2023-10-01 at 01 02 20](https://github.com/lunarphp/lunar/assets/26385667/576fc65e-46de-41f8-8558-9f4e4e200c67)

**Comment by glennjacobs** [comment]
Do you need to support multiple languages or just one? If one, then I believe you can just change the label itself.

**Reply by glennjacobs** [reply]
I'm sure if you did something like the following it would be fine...

```php
'statuses' => [
    'awaiting-payment' => [
        'label' => __('Awaiting Payment'),
```

**Reply by gausoft** [reply]
Thanks

**Comment by gausoft** [comment]
I need to handle multiple languages
